src/check.h.in

```diff
@@ -170,6 +170,23 @@ CK_DLL_EXP void CK_EXPORT suite_add_tcase(Suite * s, TCase * tc);
 CK_DLL_EXP TCase *CK_EXPORT tcase_create(const char *name);
 
 /**
+ * Create a test case associated with certain tags.
+ *
+ * Once created, tests can be added with the tcase_add_test()
+ * function, and the test case assigned to a suite with the
+ * suite_add_tcase() function.
+ *
+ * @param name name of the test case
+ * @param tags string containing arbitrary tags separated by spaces
+ *
+ * @return test case containing no tests
+ *
+ * @since 0.10.0
+ * */
+CK_DLL_EXP TCase *CK_EXPORT tcase_create_tagged(const char *name,
```

Comment:
OK, I think I am fine with passing a string that contains multiple tags which are space delimited. 
What would be expected if a user invoked the call multiple times? Namely
```
tcase_add_tags(tc, "A B C");
tcase_add_tags(tc, "D");
```

Should the listed tags be: [A B C D]?